pkg/neotest/signer.go

```diff
@@ -0,0 +1,101 @@
+package neotest
+
+import (
+	"bytes"
+
+	"github.com/nspcc-dev/neo-go/pkg/config/netmode"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/hash"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neo-go/pkg/vm/opcode"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+)
+
+// Signer is a generic interface which can be either simple- or multi-signature signer.
+type Signer interface {
```

Comment:
If it's an interface, it should hide the difference between multi-signing and single-signing.